R/fortify-network.R

```diff
@@ -199,7 +199,6 @@ fortify.network <- function(model, data = NULL,
     # add missing columns to nodes data
     nodes$xend = nodes$x
     nodes$yend = nodes$y
-    names(nodes) = names(edges)[1:ncol(nodes)]
```

Comment:
@mcanouil This line is the only one that I did not get straight away in your PR. No idea why it's not needed any more, I'll just trust you on that :)

Reply:
If I recall properly, it was something about the fact that nodes cannot have more columns than the number of edges (but it's been a while and a lot of the packages on which ggnetwork relies on have changed).